src/EthersSafe.ts

```diff
@@ -237,6 +253,106 @@ class EthersSafe implements Safe {
     return ownersWhoApproved
   }
 
+  /**
+   * Returns the Safe transaction to enable a Safe module.
+   *
+   * @param moduleAddress - The desired module address
+   * @returns The Safe transaction ready to be signed
+   */
+  async getEnableModuleTx(moduleAddress: string): Promise<SafeTransaction> {
+    const tx = new SafeTransaction({
+      to: this.getAddress(),
+      value: '0',
+      data: await this.#moduleManager.encodeEnableModuleData(moduleAddress),
+      nonce: (await this.#contract.nonce()).toNumber()
```

Comment:
nitpick: I can see at some time in the future a feature request for a way to set a custom nonce. No need to address it now but better to keep it in mind when designing the project.

Reply:
This was implemented in the other PR with some improvements. (Not pushed yet) :)